lib/error.ex

```diff
@@ -2,6 +2,18 @@ defprotocol AshGraphql.Error do
   def to_error(exception)
 end
 
+defimpl AshGraphql.Error, for: Ash.Error.Changes.InvalidChanges do
```

Comment:
I noticed UnknownError was also missing -- not sure if these are intentional or not.

Reply:
Yeah, we don't want to automatically surface unknown errors up to the user (because they could contain arbitrary data)